ttnctl/cmd/users.go

```diff
@@ -0,0 +1,145 @@
+// Copyright © 2016 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/howeyc/gopass"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type token struct {
+	AccessToken      string `json:"access_token"`
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+}
+
+// usersCmd represents the users command
+var usersCmd = &cobra.Command{
+	Use:   "user",
+	Short: "Show the current user",
+	Long:  `ttnctl user shows the current logged on user`,
+	Run: func(cmd *cobra.Command, args []string) {
+		t, err := util.LoadAuth(viper.GetString("ttn-account-server"))
+		if err != nil {
+			ctx.WithError(err).Fatal("Failed to load authentication")
+		}
+
+		if t == nil {
+			ctx.Warn("No login found")
+			return
+		}
+
+		// TODO: Validate token
+
+		ctx.Infof("Logged on as %s", t.Email)
+	},
+}
+
+var usersCreateCmd = &cobra.Command{
+	Use:   "create [e-mail]",
+	Short: "Create a new user",
+	Long:  `ttnctl user create allows you to create a new user`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			ctx.Fatal("Insufficient arguments")
+		}
+
+		email := args[0]
+		fmt.Print("Password: ")
+		password, err := gopass.GetPasswd()
+		if err != nil {
+			ctx.Fatal("Invalid password")
```

Comment:
Errors from `gopass.GetPasswd()` usually mean an interrupt or read error. I don't think "Invalid password" is a clear description of the error here...